test_macros/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "test_macros"
+version = "0.1.0"
+edition = "2024"
+
+[dependencies]
```

Comment:
is this a separate workspace to keep the dependencies separate?

Reply:
no, it's a separate crate. One limitation of Rust macros, this type of macro at least, is that you need to declare them in a separate crate to be able to use them.